src/update.cpp

```diff
@@ -411,115 +415,14 @@ void mobile_update() {
  * Update the weather.
  */
 void weather_update() {
-    char buf[MAX_STRING_LENGTH];
-    Descriptor *d;
-    int diff;
-
-    buf[0] = '\0';
-
-    switch (++time_info.hour) {
-    case 5:
```

Comment:
I've chosen to have the weather update use this cycle:

5am: light (twilight I guess?
6am: rise
7pm: set (dusk?)
8pm: night

Reply:
seems fair